package/ios/Frame Processor/FrameHostObject.mm

```diff
@@ -94,6 +151,30 @@
     };
     return jsi::Function::createFromHostFunction(runtime, jsi::PropNameID::forUtf8(runtime, "decrementRefCount"), 0, decrementRefCount);
   }
+
+  // Conversion methods
+  if (name == "getPlatformBuffer") {
+    auto getPlatformBuffer = JSI_FUNC {
+      // Lock Frame so it cannot be deallocated while we access it
+      std::lock_guard lock(this->_mutex);
+
+      // Box-cast to uintptr (just 64-bit address)
+      CMSampleBufferRef buffer = this->frame.buffer;
+      uintptr_t pointer = reinterpret_cast<uintptr_t>(buffer);
+      jsi::HostFunctionType deleteFunc = [=](jsi::Runtime& runtime, const jsi::Value& thisArg, const jsi::Value* args,
+                                             size_t count) -> jsi::Value {
+        // no-op
```

Comment:
no     CFRelease(buffer); here?

Reply:
No, I agree that this might be a bit confusing but in my case the Frame is only valid as long as the `useFrameProcessor` didn't return yet. And I `.dispose()` the SkImage before it returns, so there's no way to use it outside of it's lifetime.

Originally I wanted to use HostObjects or NativeState for this so it is properly ref-counted, but that has a few downsides:
1. It requires explicit typings, so react-native-vision-camera needed to depend on react-native-skia in native code, or vice versa so I can use the C++ types.
2. It adds overhead since we'd be using shared_ptrs. (Locks/Mutexes + Ref count increments)
3. It is unnecessary since the Frame cannot live longer than the `useFrameProcessor` anyways (and shouldn't, because that'd stall the Camera pipeline)